code/modules/surgery/organs/external/tails.dm

```diff
@@ -117,11 +117,14 @@
 	///A reference to the paired_spines, since for some fucking reason tail spines are tied to the spines themselves.
 	var/obj/item/organ/external/spines/paired_spines
 
-/obj/item/organ/external/tail/lizard/Insert(mob/living/carbon/reciever, special, drop_if_replaced)
+//BUBBER FIX START
+//This edit atempts to bandaid the spine wagging issue and should be overriden the moment someone else figures it out
+/obj/item/organ/external/tail/lizard/Insert(mob/living/carbon/receiver, special, drop_if_replaced)
 	. = ..()
-	if(.)
-		paired_spines = ownerlimb.owner.get_organ_slot(ORGAN_SLOT_EXTERNAL_SPINES)
-		paired_spines?.paired_tail = src
+	paired_spines = ownerlimb.owner.get_organ_slot(ORGAN_SLOT_EXTERNAL_SPINES)
```

Comment:
The animation on these is still broken but I didn't notice (Spines disappear now), Soothes thankfully brought it up.

Will fix asap

Reply:
Mind you this is still better than them not animating

Reply:
Ok I looked through this. The `build_icon_state()` now works correctly, updating the icon state... but something still makes the spines disappear????